src/plugins/requestAuthPlugin.js

```diff
@@ -244,13 +250,12 @@ const refreshToken$ = refresh_token => Rx.Observable.of({
  * For an anonymous auth, the request header information is used to determine
  * the location and language of the anonymous member
  *
- * @param {Object} config { OAUTH_AUTH_URL, OAUTH_ACCESS_URL, oauth }
  * @param {Object} request the Hapi request that needs to be authorized
  */
-export const getRequestAuthorizer$ = config => {
+export const getRequestAuthorizer$ = () => {
```

Comment:
I'd prefer to keep existing pure functions pure, i.e. making sure that we pass in the data they need as arguments rather than giving them having an implicit reliance on scope provided by an `import`. The downside is that we have to thread configuration down to various functions, but that's far better IMO than a `config` import that can lead to spaghetti code with implicit dependencies.

The original code wasn't as specific as it could/should be (e.g. passing the whole config object when functions only needed a subset of properties), but I'd lean toward fixing that weakness rather than doubling down :)